### PR TITLE
Allow Rails 7.2

### PR DESCRIPTION
**Pull request by texpert**
Add `7-2-stable` Rails branch to the CI

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         ruby: ["3.0", "3.1", "3.2", "3.3"]
-        appraisal: [rails_6_0, rails_6_1, rails_7_0, rails_7_1]
+        appraisal: [rails_6_0, rails_6_1, rails_7_0, rails_7_1, rails_7_2]
```
Comment (mvz) [inline .github/workflows/ruby.yml:24]: From the test failures, it looks like rails 7.2 will not support Ruby 3.0. Can you please exclude that combination from the matrix?

File: non-digest-assets.gemspec [hunk 2]
```diff
@@ -22,13 +22,13 @@ Gem::Specification.new do |spec|
 
   spec.metadata["rubygems_mfa_required"] = "true"
 
-  spec.add_dependency "activesupport", [">= 6.0", "< 7.2"]
+  spec.add_dependency "activesupport", [">= 6.0", "<= 7.2"]
```
Comment (mvz) [inline non-digest-assets.gemspec:25]: This will exclude activesupport 7.2.1 and up. Please change it to `< 7.3`.

File: non-digest-assets.gemspec [hunk 3]
```diff
@@ -22,13 +22,13 @@ Gem::Specification.new do |spec|
...
   spec.add_dependency "sprockets", "~> 4.0"
 
   spec.add_development_dependency "appraisal", "~> 2.3"
   spec.add_development_dependency "aruba", "~> 2.0"
   spec.add_development_dependency "pry", "~> 0.14.0"
-  spec.add_development_dependency "rails", [">= 6.0", "< 7.2"]
+  spec.add_development_dependency "rails", [">= 6.0", "<= 7.2"]
```
Comment (mvz) [inline non-digest-assets.gemspec:31]: This will exclude rails 7.2.1 and up. Please change it to `< 7.3`.

File: gemfiles/rails_7_2.gemfile [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "rails", git: 'https://github.com/rails/rails', branch: '7-2-stable'
```
Comment (mvz) [inline gemfiles/rails_7_2.gemfile:7]: Please fix the RuboCop offenses on this line.

**Review by mvz — CHANGES_REQUESTED** [review]
Thanks for this @texpert! If you can take care of my comments this should be good to merge.

**Comment by texpert** [comment]
> Thanks for this @texpert! If you can take care of my comments this should be good to merge.

Thanks for review, @mvz! I have addressed all the comments.

**Review by mvz — APPROVED** [review]

**Comment by mvz** [comment]
Thanks again, @texpert!

**Comment by texpert** [comment]
Thanks for help, @mvz!

Can you, please, craft a new release?!

**Comment by mvz** [comment]
Hi @texpert I just released version 2.4.0 which includes this change.

**Pull request merged**